cmd/frontend/internal/httpapi/src_cli.go

```diff
@@ -0,0 +1,61 @@
+package httpapi
+
+import (
+	"net/http"
+	"path"
+
+	"github.com/gorilla/mux"
+	srccli "github.com/sourcegraph/sourcegraph/internal/src-cli"
+	log15 "gopkg.in/inconshreveable/log15.v2"
+)
+
+var srcCliDownloadsURL = "https://github.com/sourcegraph/src-cli/releases/download"
+
+var whitelistedFilenames = []string{
+	"src_darwin_amd64",
+	"src_linux_amd64",
+	"src_windows_amd64.exe",
+}
+
+func srcCliVersionServe(w http.ResponseWriter, r *http.Request) error {
+	return writeJSON(w, &struct {
+		Version string `json:"version"`
+	}{
+		Version: srcCliVerison(),
```

Comment:
```suggestion
		Version: srcCliVersion(),
```

Why is Verizon here? 😛

Reply:
Fixed.